instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -2,43 +2,99 @@ import React from "react";
 import "./SearchBar.css";
 import cam from "../../assets/instagram-brands.svg";
 import textLogo from "../../assets/insta-text-logo.svg";
-import styles from "styled-components"
-
-const Header = styles.header`
-border-bottom: 1px solid #3333;
-height: 80px;
-background: #ffffff;
-position: fixed;
-width: 100vw;
-top: 0;
+import styled from "styled-components"
+
+const Header = styled.header`
```

Comment:
I can see why this gave you a lot to move around. It would have been better to import them from another file. Good work on your progress! Keep it up. :)